### ISSUE TITLE
Allow mariadb to work with 10.3.x and up

### ISSUE THREAD

**Issue by stronk7** [moodlehq/moodle-docker#93]
In #25 we unpinned mariadb back to 10... (right now pointing to 10.3.x)... just to discover that there are some config variables not existing anymore:

```
2019-01-18 22:25:48 0 [ERROR] mysqld: unknown variable 'innodb_file_format=barracuda'
2019-01-18 22:25:48 0 [ERROR] Aborting
```

Preventing the server to startup. So we pinned it back to 10.2.

This issue is about to try mariadb compatible with all the versions. Then we'll be really able to unpin it completely.

Ciao :-)

**Comment by deer667** [comment]
In MariaDB 10.2.2 and later, the default file format is Barracuda and Antelope is deprecated.

The config var can be removed and never verison of MariaDB can be pinned.

**Comment by scara** [comment]
Hi @deer667,
could you provide a ref for your statement?

I tried both https://mariadb.com/kb/en/mariadb-1022-release-notes/ and https://mariadb.com/kb/en/mariadb-1022-changelog/ w/o success.

TIA,
Matteo

**Comment by stronk7** [comment]
Fixed by #134, so far, closing.
